Yona/Yona/Settings/SettingsViewController.swift

```diff
@@ -110,7 +122,12 @@ extension SettingsViewController:UITableViewDelegate {
     }
     
     func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
-        if indexPath.row == 2 {
+        if indexPath.row == 0 {
+            //change pin
+        } else if indexPath.row == 1 {
+            //privacy
+            performSegueWithIdentifier("privacyStatement", sender: self)
+        } else if indexPath.row == 2 {
             callAddDeviceMethod()
         }
         else if indexPath.row == 3 {
```

Comment:
Enum will definitely solve the readability issue, but you will still be tightly coupled with the order which can easily change when you added or removed other items to the list.